### PR TITLE
add badge for service

### PR DESCRIPTION
**Pull request by gaocegege**
Signed-off-by: Ce Gao <ce.gao@outlook.com>

### CODE REVIEW

**Comment by gaocegege** [comment]
PTAL @superxi911 @xiaomuqiao 
This feature needs to re-deploy cyclone. And the badge would return the latest version status.

**Comment by xiaomuqiao** [comment]
LGTM

**Comment by xiaomuqiao** [comment]
The next is that testing with cyclone web.

**Comment by superxi911** [comment]
LGTM

File: api/rest/service.go [hunk 1]
```diff
@@ -27,6 +27,14 @@ import (
...
+
+	// failureBadge is the badge showed when version creation is failed.
+	failureBadge = `<svg xmlns="http://www.w3.org/2000/svg" width="80" height="20"><linearGradient id="b" x2="0" y2="100%"><stop offset="0" stop-color="#bbb" stop-opacity=".1"/><stop offset="1" stop-opacity=".1"/></linearGradient><mask id="a"><rect width="80" height="20" rx="3" fill="#fff"/></mask><g mask="url(#a)"><path fill="#555" d="M0 0h37v20H0z"/><path fill="#e05d44" d="M37 0h43v20H37z"/><path fill="url(#b)" d="M0 0h80v20H0z"/></g><g fill="#fff" text-anchor="middle" font-family="DejaVu Sans,Verdana,Geneva,sans-serif" font-size="11"><text x="18.5" y="15" fill="#010101" fill-opacity=".3">build</text><text x="18.5" y="14">build</text><text x="57.5" y="15" fill="#010101" fill-opacity=".3">failing</text><text x="57.5" y="14">failing</text></g></svg>`
```
Comment (xiaomuqiao) [inline api/rest/service.go]: i think  cyclone only needs to provide results, how display the result  can be  implemented by a third party. Just a   little idea.

Comment (gaocegege) [inline api/rest/service.go]: We don't have third party support now and we should have a solution now, so I think we could keep it now.

**Pull request merged**